src/database/utilities/createRecord.js

```diff
@@ -647,7 +647,8 @@ const createTransactionItem = (database, transaction, item, initialQuantity = 0)
     transaction,
   });
 
-  transactionItem.setTotalQuantity(database, initialQuantity);
+  const { isFinalised } = transaction;
```

Comment:
you can add items to a finalised transaction? What am I missing here?

Reply:
sync - before this change it would try set the total quantity to 0 initially for transaction items of a transaction that came in from sync

Reply:
ah! that makes sense, thanks.